pkg/core/blockchain.go

```diff
@@ -2085,7 +2091,7 @@ func (bc *Blockchain) PoolTxWithData(t *transaction.Transaction, data interface{
 
 // GetStandByValidators returns validators from the configuration.
 func (bc *Blockchain) GetStandByValidators() keys.PublicKeys {
-	return bc.sbCommittee[:bc.config.ValidatorsCount].Copy()
+	return bc.sbCommittee[:bc.config.GetNumOfCNs(bc.BlockHeight())].Copy()
```

Comment:
I'm not sure, but currently, the whole set of committee members from config is stored in `bc.sbCommittee` irrespective to committee update history. Consequently, the whole set of values is returned from the `bc.GetStandByCommittee` method. Native NEO contract uses `bc.GetStandByCommittee` for initialisation (`(n *NEO) Initialize). As a result, during initialisation committee hash is calculated using all the provided committee members. Is it correct?

Reply:
I don't think `getcommittee` is affected, it uses `GetCommittee` which works fine.

Reply:
OK, it's clearer now with the last commit.